components/prism-elm.js

```diff
@@ -0,0 +1,35 @@
+Prism.languages.elm = {
+	comment: {
+		pattern: /(^|[^-!#$%*+=?&@|~.:<>^\\\/])(--[^-!#$%*+=?&@|~.:<>^\\\/].*|{-[\s\S]*?-})/m,
+		lookbehind: true
+	},
+	char: /'([^\\']|\\([abfnrtv\\"'&]|\^[A-Z@[\]\^_]|NUL|SOH|STX|ETX|EOT|ENQ|ACK|BEL|BS|HT|LF|VT|FF|CR|SO|SI|DLE|DC1|DC2|DC3|DC4|NAK|SYN|ETB|CAN|EM|SUB|ESC|FS|GS|RS|US|SP|DEL|\d+|o[0-7]+|x[0-9a-fA-F]+))'/,
+	string: {
+		pattern: /"([^\\"]|\\([abfnrtv\\"'&]|\^[A-Z@[\]\^_]|NUL|SOH|STX|ETX|EOT|ENQ|ACK|BEL|BS|HT|LF|VT|FF|CR|SO|SI|DLE|DC1|DC2|DC3|DC4|NAK|SYN|ETB|CAN|EM|SUB|ESC|FS|GS|RS|US|SP|DEL|\d+|o[0-7]+|x[0-9a-fA-F]+)|\\\s+\\)*"/,
```

Comment:
Same comment here: this looks overcomplicated. Are there string-like patterns in Elm that are not to be highlighted as `string`?

Reply:
You're right, this is way overcomplicated. I adapted it from the haskell one; and I'm now realizing that - other than being overcomplicated - it's also wrong. Elm doesn't do multiline strings that way, but rather with `"""`. Trying to figure out a good way to match either simple strings or multiline strings, which is slightly complicated by the latter being able to include single `"`.